jobs/WebUi/MasterCI/MasterCI

```diff
@@ -0,0 +1,60 @@
+node{
```

Comment:
As we talk, we are heading for a SprintRelease pipeline as first priority, instead of  MasterCI purpose.
if you think your code can be reusable for both MasterCI & Sprint Release, then we can pick up another  file name ?